homeassistant/components/squeezebox/media_player.py

```diff
@@ -587,4 +588,18 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
             "search_id": media_content_id,
         }
 
-        return await build_item_response(self._player, payload)
+        return await build_item_response(self, payload)
+
+    async def async_get_browse_image(
+        self, media_content_type, media_content_id, browse_image
+    ):
+        """Get album art from Squeezebox server."""
+        image_url = self._player._lms.generate_image_url_from_track_id(browse_image)
```

Comment:
@rajlaud any chance we could make this a public method on the _player that delegates down to _lms internally?